feconf.py

```diff
@@ -991,3 +991,90 @@ def get_empty_ratings():
 
     }
 }
+
+# Validation criteria for customization arguments of rich text components
+# along with their default values.
+RICH_TEXT_COMPONENTS = [{
```

Comment:
Nope -- there should be a single source of truth for this. Don't maintain two declarative versions of the same thing. Instead, find a way to derive this on-the-fly from an existing source of truth, e.g. assets/rich_text_components_definitions.js.

Reply:
Done.